apps/frontend/package.json

```diff
@@ -21,17 +29,18 @@
     "@headlessui/react": "^1.7.17",
     "@heroicons/react": "^2.0.18",
     "@lezer/common": "^1.1.0",
+    "@monaco-editor/react": "^4.6.0",
     "@types/node": "^20.6.3",
     "@types/react": "^18.2.22",
     "@types/react-dom": "^18.2.7",
     "@uiw/codemirror-theme-material": "^4.21.18",
     "@uiw/react-codemirror": "^4.21.18",
     "axios": "^1.5.0",
-    "codemirror": "^6.0.1",
```

Comment:
was this removed accidentally? or is this not needed anymore?

Reply:
oh shute i removed it and forgot to add it back my bad

Reply:
Fixed!